### PR TITLE
Fix default elasticpress defines

### PR DESCRIPTION
**Pull request by mikelittle**
ElasticPress plugin expects EP_HOST to be a full url including port number. 
Addresses https://github.com/humanmade/platform-dev/issues/105

### CODE REVIEW

File: lib/elasticpress-integration.php [hunk 1]
```diff
@@ -10,7 +10,9 @@ function bootstrap() {
 	if ( ! defined( 'ELASTICSEARCH_HOST' ) ) {
 		return;
 	}
-	define( 'EP_HOST', ELASTICSEARCH_HOST );
+	if ( ! defined( 'EP_HOST' ) ) {
+		define( 'EP_HOST', sprintf( 'http://%s:%s', ELASTICSEARCH_HOST, ELASTICSEARCH_PORT ) );
```
Comment (joehoyle) [inline lib/elasticpress-integration.php]: This should be `https` if the port is 443 IIRC, see http://docs.aws.hmn.md/beta/cloud/components/elasticsearch/

Comment (mikelittle) [reply lib/elasticpress-integration.php]: The code a couple of lines down (lines 17 and 18)  seems to hard code it to https anyway. Which presumably is true regardless of the port number on the real platform. But working locally, which still uses the default ELASTICSEARCH_PORT setting, it won't be https.

Comment (joehoyle) [reply lib/elasticpress-integration.php]: Either way I don't think we want this defined as such. It's a constant, so can never be changed, we should therefor make it correct. I see no problem not to.

> But working locally, which still uses the default ELASTICSEARCH_PORT setting, it won't be https.

Then have it use `http` when it's not port 443.

Comment (joehoyle) [reply lib/elasticpress-integration.php]: To clarify on the above, `ELASTICSEARCH_PORT` will not be 443 locally, unless you _are_ using https. or should be that way anywhere.

**Pull request merged**